databallpy/utils/synchronise_tracking_and_event_data.py

```diff
@@ -248,16 +266,24 @@ def _needleman_wunsch(
             t[0] = F[i, j] + sim_mat[i, j]
             t[1] = F[i, j + 1] + gap_frame  # top + gap frame
             t[2] = F[i + 1, j] + gap_event  # left + gap event
-            tmax = np.max(t)
-            F[i + 1, j + 1] = tmax
 
-            if t[0] == tmax:  # match
+            # manually calculate tmax instead of using np.max() since it is
```

Comment:
This helped me understand but perhaps we should not keep it there as no one knows what the code looked like previously so perhaps it is confusing to add reference to the old code

Reply:
I think it would be better to leave it in since it's the pythonic way to do `np.max`, so explaining why we do it this way has added value, also for the future.